File: GTTS/INL.VettingService.Models/GetCourtesyBatch_Result.cs
using System;
using System.Collections.Generic;
using System.Text;
using INL.Services.Models;

namespace INL.VettingService.Models
{
    public class GetCourtesyBatch_Result : BaseResult, IGetCourtesyBatch_Result
    {
        public ICourtesyBatch_Item item { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-courtesy-batch_result.ts


import { ICourtesyBatch_Item } from './icourtesy-batch_item';
import { IGetCourtesyBatch_Result } from './iget-courtesy-batch_result';

export class GetCourtesyBatch_Result implements IGetCourtesyBatch_Result {
  
	public item?: ICourtesyBatch_Item;
  
	public ErrorMessages: string[] = null;
}


